packages/libs/src/sdk/sdk.ts

```diff
@@ -293,7 +293,7 @@ const initializeApis = ({
     services.logger
   )
   const tips = new TipsApi(generatedApiClientConfig)
-  const { resolve } = new ResolveApi(generatedApiClientConfig)
+  const resolve = new ResolveApi(generatedApiClientConfig)
```

Comment:
we would have to spread out the whole resolve API onto the root to make that work because the `this` context of the resolve that's there is just sdk root. I think this is fine. maybe we add `resolveTrack` in the future or something and in that case, it makes sense to treat the same as other APIs

Reply:
Wouldn't this work?
```ts
const resolveApi = new ResolveApi(..)
const resolve = resolveApi.resolve.bind(resolveApi)
// ...
return {
    /// ...
    resolve,
}
```

But eh, not super important

Reply:
oh yeah that would work if done here. i can dig it so we don't change the interface and this is not a significant version bump